packages/wall/wall.0.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "Frederic Bour <frederic.bour@lakaban.net>"
+authors: "Frederic Bour <frederic.bour@lakaban.net>"
+homepage: "https://github.com/let-def/wall"
+bug-reports: "https://github.com/let-def/wall"
+license: "BSD3"
+dev-repo: "git+https://github.com/let-def/wall.git"
+build: [
+  ["jbuilder" "build" "--only" "wall" "--root" "." "-j" jobs "@install"]
+]
+depends: [
+  "ocamlfind" {build}
```

Comment:
You seem to use jbuilder but you forgot to add it as dependency. Also are you still using `ocamlfind` ?